deploy/crds/devopsconsole_v1alpha1_installer_cr.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: devopsconsole.openshift.io/v1alpha1
+kind: Installer
+metadata:
+  name: main-installer
+spec: {}
```

Comment:
Can we modify the CRD to support listing which service is to be installed?

At the CR, we could specify the name of the service's image's environment variable example "APP_SERVICE" , and the controller could simply use that information to get the image name?

Reply:
The above comment is intended to help add multiple services - and to do so all someone would needs to do is:

- add environment variable to the operator with the image name.
- create a new installer CR referring to the image name by the environment variable.

Reply:
>At the CR, we could specify the name of the service's image's environment variable example "APP_SERVICE" , and the controller could simply use that information to get the image name?

What would happen if the `APP_SERVICE` entry was removed from the CR? Should the deployment, service, routes should also be deleted?

```yaml
spec:
  services:
    - name: "app-service"
    - name: "another-service"
```
Would the new yaml look like this?

Reply:
> What would happen if the APP_SERVICE entry was removed from the CR? Should the deployment, service, routes should also be deleted?

Yes.


> Would the new yaml look like this?

Yes. Though you need to find a way to find a connection between the "app-service" and the environment variable that holds the image for that.

Reply:
We should do this in a separate PR. This PR is already huge.

Reply:
Maybe we can create a separate story for this? It looks like this story has already ballooned from the original description in the story. We should probably try to avoid doing this as it messes up our estimates in terms of how much work we planned the story to be vs how much work it actually turned out to be.